package.json

```diff
@@ -22,14 +22,13 @@
     "generate": "stencil generate"
   },
   "dependencies": {
-    "@stencil/core": "^2.0.1"
+    "@stencil/core": "^2.7.0"
```

Comment:
This wasn't technically needed, but I wanted to show the association between the new version of Stencil and the new Puppeteer version